### DISCUSSION TITLE
Ubuntu desktop vs Ubuntu server for a large camera deployment

### DISCUSSION THREAD

**Discussion by Sceptersax** [blakeblackshear/frigate#7753] [Ask A Question] [answered]
Probably a dumb question here, but a friend of mine recommended that I go with Ubuntu server for the frigate build I'm doing. I never used it. Then I find out there's no GUI... which makes sense i guess. Eliminating Gnome and other graphical user processes would streamline things, and since I plan on scaling the camera system to a very large deployment, we need every bit of resource we can get. But is this a recommended method? I suppose the docker containers' webservers can be accessed remotely with a client pc in the same network and a browser can also be accessed on the client pc... But Is there anything I could be missing out on if I decide to go Ubuntu server vs desktop? 

note: I plan on running: frigate, HA, double-take and other detectors. Besides that, nothing else.

**Comment by NickM-27** [comment, marked as answer]
I don't think there are any issues with running a headless OS, main thing is that you'll need to install the GPU drivers manually (most likely)

**Reply by Sceptersax** [reply]
Good to know. I'll give it a shot then. Thank you.

**Comment by Cheerpipe** [comment]
Hi.

I am running frigate in Ubuntu server in a headless machine. It works well, including hw acceleration (Intel and Nvidia) for video decoding. Openvino and Nvidia detectors also works as expected. 

Double Take and GPU accelerated Compreface works too.

**Reply by Sceptersax** [reply]
Thanks! I’m trying it now.
